src/AdminConsole/Pages/Organization/Create.cshtml.cs

```diff
@@ -19,6 +19,7 @@ public class Create : PageModel
     private readonly MagicLinkSignInManager<ConsoleAdmin> _magicLinkSignInManager;
     private readonly IEventLogger _eventLogger;
     private readonly ILogger<Create> _logger;
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
Where is this referenced in code? I can't see it being used 🤔

Reply:
Its not :D